src/app/services/tasksApi.ts

```diff
@@ -1,14 +1,27 @@
-import task from '@/interfaces/task.type';
+import task, { updateTaskDetails } from '@/interfaces/task.type';
 import { api } from './api';
-import { MINE_TASKS_URL } from '@/constants/url';
+import { MINE_TASKS_URL, TASKS_URL } from '@/constants/url';
+TASKS_URL;
```

Comment:
Nit: I think it will get removed

Reply:
Thanks for pointing this out. Removed it.